test_runner/batch_others/test_wal_acceptor.py

```diff
@@ -314,14 +316,23 @@ def test_race_conditions(zenith_env_builder: ZenithEnvBuilder, stop_value):
     proc.join()
 
 
-class ProposerPostgres:
-    """Object for running safekeepers sync with walproposer"""
-    def __init__(self, env: ZenithEnv, pgdata_dir: str, pg_bin, timeline_id: str, tenant_id: str):
-        self.env = env
+class ProposerPostgres(PgProtocol):
+    """Object for running postgres without ZenithEnv"""
+    def __init__(self,
+                 pgdata_dir: str,
+                 pg_bin,
+                 timeline_id: str,
+                 tenant_id: str,
+                 listen_addr: str,
+                 port: int):
+        super().__init__(host=listen_addr, port=port)
+
         self.pgdata_dir: str = pgdata_dir
```

Comment:
Tried to replace all paths with pathlib.Path in a file, look at https://github.com/zenithdb/zenith/commit/211695ab9d652eb722d7dfa33a3f44f0becfac5f. Not sure if it's better because we still convert them to str later and it's easy to mix up str with pathlib and misuse them.

Reply:
We need strings only when paths are passed as some command arguments, so this should be ok to have a conversion in argument list. Also we have mypy, so it is less likely to have confusion between the two